controllers/ovndbcluster_controller.go

```diff
@@ -581,6 +590,76 @@ func (r *OVNDBClusterReconciler) reconcileServices(
 			}
 		}
 	}
+
+	Log.Info("DNS Starts here")
+	if instance.Spec.DBType == v1beta1.SBDBType {
+		Log.Info("Only doing it for SB pods")
```

Comment:
why? NB is consumed by OVN Agent and I think BGP agent also plans to switch to NB.

Reply:
I think both should be exposed if attached to NAD, and the operator should not care about which agents are running. (In theory, the operator could be used outside of OpenStack env - at least it's my belief that we should not assume OpenStack.)

Reply:
Extended also to NB